### PR TITLE
[jsk_apc2016_common] update rbo_segmentation submodule

### PR DESCRIPTION
**Pull request by yuyu2172**
Now follow jsk_apc branch
https://github.com/yuyu2172/rbo-apc-object-segmentation

The branch is also updated. Now data class APCData accepts matrix as input.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,3 +4,4 @@
 [submodule "jsk_apc2016_common/python/jsk_apc2016_common/rbo_segmentation"]
 	path = jsk_apc2016_common/python/jsk_apc2016_common/rbo_segmentation
 	url = https://github.com/yuyu2172/rbo-apc-object-segmentation
+  branch = jsk_apc
```
Comment (wkentaro) [inline .gitmodules]: indent.

**Comment by wkentaro** [comment]
Please use refine commit message. 
algorithm's -> rbo_segmentation?

**Comment by yuyu2172** [comment]
fixed

**Comment by wkentaro** [comment]
+1

On Thu, Apr 28, 2016 at 7:53 PM, Yusuke Niitani <notifications@github.com>
wrote:

> fixed
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/start-jsk/jsk_apc/pull/1359#issuecomment-215386728>
>



-- 
和田 健太郎 / Kentaro Wada
http://wkentaro.com

**Pull request merged**